toast-tk-web-agent/src/main/java/io/toast/tk/agent/web/IAgentServer.java

```diff
@@ -2,12 +2,14 @@
 
 import io.toast.tk.core.agent.interpret.WebEventRecord;
 
+import java.net.UnknownHostException;
+
 public interface IAgentServer {
 
 	void sendEvent(WebEventRecord eventRecord, String ApiKey);
 
 	boolean register(String ApiKey);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00117)